packages/inertia/src/events.js

```diff
@@ -9,7 +9,11 @@ export function fireBeforeEvent(visit) {
 }
 
 export function fireErrorEvent(error) {
```

Comment:
I think plural here makes more sense.
```suggestion
export function fireErrorEvent(errors) {
```